rust_src/remacs-sys/lib.rs

```diff
@@ -700,6 +700,7 @@ extern "C" {
     pub static Qt: Lisp_Object;
     pub static Qarith_error: Lisp_Object;
     pub static Qnumber_or_marker_p: Lisp_Object;
+    pub static Qinteger_or_marker_p: Lisp_Object;
```

Comment:
Sooner or later we should look at defining built-in symbols in Rust. It's the main thing stopping us writing useful unit tests.